src/aui-button/js/aui-button-core.js

```diff
@@ -217,6 +249,17 @@ ButtonExt.prototype = {
     },
 
     /**
+     * Fires on `visibleChange` event
+     *
+     * @method _onVisibleChange
+     * @param value {EventFacade} event
+     * @protected
+     */
+    _onVisibleChange: function(event) {
+        this._uiSetVisibility(event.newVal);
```

Comment:
We always name these kind of functions in the format `_uiSet<AttrName>`, so this should be `_uiSetVisible` to follow alloy-ui's standards.